log.go

```diff
@@ -84,6 +89,32 @@ type Logger interface {
 	Fatalw(msg string, fields Fields)
 }
 
+// Atoi returns the loglevel integer associated with a common loglevel
+// string representation.
+func Atoi(loglevelStr string) (level int, err error) {
```

Comment:
Instead of calling it `Atoi` can the function be named `ParseLevel`?

While the implementations of the levels are ints under the hood, the ints aren't really used as ints. `Atoi` is a common function name for a function that turns strings into ints. This is turning specific names into Levels.

With that in mind, can a new type be created for the Level and return that instead of an int.

Reply:
Hi Matt, thanks for the feedback.  Regarding the use case for the function: My goal was to provide a means for translating the string-type loglevels usually specified in config files into the internal log-go loglevels.  This could be done at the downstream implementation but, I think, the string representations of log levels are sufficiently standardised that the upstream library is the most logical place to host this function.